src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,10 @@
+$c-main: #060b35;
+$c-blue: #00acdc;
+$c-secondary: #616070;
+$c-white: #fff;
```

Comment:
maybe better  stick to some style) i think $c-blue and $c-secondary it's different styles)